docs/application/dotnet/guides/nui/text.md

```diff
@@ -401,6 +401,23 @@ The following example illustrates text alignment:
 field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
+<a name="input-properties"></a>

```

Comment:
FYI (optional) : If there is no place which is using the name "input-properties", we can skip adding the name.

Reply:
Oh, Then I'll skip adding the name here. Thank you for looking at it again.